rundeckapp/grails-app/views/execution/_wfstateSummaryLine.gsp

```diff
@@ -32,7 +32,7 @@
     </span>
     <span class="timerel">
         <g:message code="at" />
-        <span data-bind="text: formatTimeAtDate(endTime()), attr: {title: endTime() }">
+        <span data-bind="text: formatTimeAtDate('${execution.dateCompleted}'), attr: {title: endTime() }">
```

Comment:
not sure this is good, I think `Date.toString()` is dependent on locale, so the string might change.  I think it is better to use the javascript `endTime()`.  (also 2 other locations)